api/src/queries/project/project-view-queries.ts

```diff
@@ -3,6 +3,62 @@ import { getLogger } from '../../utils/logger';
 
 const defaultLog = getLogger('queries/project/project-view-queries');
 
+/**
+ * SQL query to get a single public (published) project.
+ *
+ * @param {number} projectId
+ * @returns {SQLStatement} sql query object
+ */
+export const getPublicProjectSQL = (projectId: number): SQLStatement | null => {
+  defaultLog.debug({ label: 'getPublicProjectSQL', message: 'params', projectId });
+
+  if (!projectId) {
+    return null;
+  }
+
+  const sqlStatement = SQL`
+    SELECT
+      project.project_id as id,
+      project_type.name as type,
+      project.name,
+      project.objectives,
+      project.location_description,
+      project.start_date,
+      project.end_date,
+      project.caveats,
+      project.comments,
+      project.coordinator_first_name,
+      project.coordinator_last_name,
+      project.coordinator_email_address,
+      project.coordinator_agency_name,
+      project.coordinator_public,
```

Comment:
Can this be public?

Reply:
not always, it depends on state of `coordinator_public`
will change this, nice catch

Reply:
I think robin mentioned something about the agency always being visible? its just the name/email that is conditional